src/vm/types.rs

```diff
@@ -761,24 +772,23 @@ impl TypeSignature {
     }
 
     fn expand_to_admit(&mut self, x_type: &TypeSignature) -> Result<()> {
-        if let Some(ref mut my_list_dimensions) = self.list_dimensions {
-            if let Some(ref x_list_dimensions) = x_type.list_dimensions {
+        use self::TypeSignature::{Atom, List};
```

Comment:
Please move this to the head of the file

Reply:
I kind of disagree with importing enum variants at the head of large files -- it can really obscure the type that is being operated on. But if you feel strongly about, I'll move it.

Reply:
The reasons I feel this way are because (1) I don't want something imported more than once since it makes refactoring difficult and (2) I don't like the meaning of a particular type to change based on which code block the import is in (e.g. I want `Atom` to be defined and have exactly one meaning throughout the whole file).  Developing the habit of putting `use` statements within code blocks works against these two goals.

Reply:
Okay, removed the `use` statement.

Reply:
Thanks!